pkg/sources/filesystem/filesystem.go

```diff
@@ -270,7 +270,7 @@ func (s *Source) Enumerate(ctx context.Context, reporter sources.UnitReporter) e
 
 // ChunkUnit implements SourceUnitChunker interface.
 func (s *Source) ChunkUnit(ctx context.Context, unit sources.SourceUnit, reporter sources.ChunkReporter) error {
-	path := unit.SourceUnitID()
+	path, _ := unit.SourceUnitID()
```

Comment:
Does the linter complain if you leave this out?

Reply:
Are you referring to leaving out the `_`? I think go won't compile it due to assignment mismatch.